src/chapters/11-route-params.md

```diff
@@ -95,7 +110,19 @@ Alright, we have just one more step left here: updating the tests. We can add an
 git add tests/integration/components/rental-test.js
 ```
 
-Awesome! However, there is one more point to mention here: by default, a component test (like the one that we have here) does not set up the router. Usually, this is what you'd want, since for more components, you likely aren't going need the router to test what you want. But in this specific case, we have a `<LinkTo>` in our component that is relying on the router to generate its URLs.
+If we run the tests in the browser, everything should...
+
+<!-- TODO: Make sure we show screenshot of failing test here. Otherwise reader might be confused that their tests are failing before they read the explanation below. -->
```

Comment:
```suggestion
```

Reply:
The change seems good to me, so I think we can just go ahead and do that and not need the comment here